awscli/examples/s3/cp.rst

```diff
@@ -151,16 +151,21 @@ Output::
 
 **Uploading a local file stream to S3**
 
-WARNING:: PowerShell may alter the encoding of or add a CRLF to piped input.
+.. WARNING:: PowerShell may alter the encoding of or add a CRLF to piped input.
 
 The following ``cp`` command uploads a local file stream from standard input to a specified bucket and key::
 
     aws s3 cp - s3://mybucket/stream.txt
 
+**Uploading a local file stream that is larger than 50gb to S3**
+
+The following ``cp`` command uploads a local file stream from standard input to a specified bucket and key.  The ``--expected-size`` option must be provided, or the upload may fail when it reaches the default part limit of 10,000::
+
+    aws s3 cp - s3://mybucket/stream.txt --expected-size <expected_size_in_bytes>
```

Comment:
It may be worth adding an actual value here instead of the placeholder `<expected_size_in_bytes>` to give an example of how the value should look, and in the prose we can mention the local file stream is the same size as that value.